docs/examples/bower.yml

```diff
@@ -2,4 +2,4 @@ excludes:
   scopes:
   - name: "devDependencies"
     reason: "BUILD_TOOL_OF"
-    comment: "These are dependencies only used for development. They are not distributed in the context of this product."
+    comment: "Scope with dependencies only used for development. Not included in released artifacts in the context of this project."
```

Comment:
optional: "Scope with" is also redundant.

I'd like it short: 

comment: "Development dependencies."

Reply:
Added "Scope" for the ORT reports for non-technical users - fine with making it shorter to just "Dependencies only used for development"